uiowa/uiowa
# Feature mega menu

Styles the Mega Menu option to expose the 3rd level menu links.

Notes:
It looks like there might be some missing JS that needs to be added. On the mobile accordion menu, the 3rd level ul's are missing the class of .sf-hidden. There is some filler css to fix this.


# How to test


- Review the code changes
  - We have refactored the menu SCSS into separate partials for maintainability.  Please test out all three menu options in the theme settings and verify that they are working as expected. 


```
ddev blt ds --site=nursing.uiowa.edu
yarn workspace uids_base build
ddev drush @nursing.local uli /admin/appearance/settings/uids_base
```
Change navigation style to Mega Menu:

<img width="327" alt="Screen Shot 2021-12-22 at 12 04 39 PM" src="https://user-images.githubusercontent.com/1036433/147136202-23fa05e2-cdff-4d08-ae51-51e55d4eba15.png">

- Try tabbing through the mega navigation and look for any accessibility issues. 

**There is an issue for the mobile view that we are hiding with CSS.  Without it, hovering over the menu shows the expanded menu like in the screenshot below:**

<img width="489" alt="Screen Shot 2021-12-21 at 11 11 16 AM" src="https://user-images.githubusercontent.com/1036433/147136380-ec110f10-6c30-4a29-a4c0-834ba502551d.png">
